webtester-core/src/main/java/info/novatec/testit/webtester/pageobjects/Select.java

```diff
@@ -300,6 +307,7 @@ public Integer execute(PageObject pageObject) {
 
             @Override
             public List<String> execute(PageObject pageObject) {
+                pageObject.markAsRead();
```

Comment:
Most of the `pageObject.markAsRead();` could be replaced by putting them into `getAllSelectedOptions()`, `getFirstSelectedOption()` and `getAllOptions()`